apis/python/setup.py

```diff
@@ -22,10 +22,15 @@
 import sys
 
 import setuptools
-
-# import setuptools.command.build_ext
+import setuptools.command.build_ext  # FIXME: why was this commented out?
 import wheel.bdist_wheel
-from pybind11.setup_helpers import Pybind11Extension
+
+try:
+    from pybind11.setup_helpers import Pybind11Extension
+except ImportError:
+    # explanation:
+    # https://pybind11.readthedocs.io/en/stable/compiling.html#classic-setup-requires
+    from setuptools import Extension as Pybind11Extension
```

Comment:
This works around a catch-22 where pybind11 is a requirement to load setup.py, yet pip cannot read&install our requirements without loading setup.py.